components/proto/src/file/app.rs

```diff
@@ -90,16 +90,32 @@ pub fn store_trusted_app_to_file(
     Ok(())
 }
 
+#[derive(Debug)]
+pub enum AppDirectoryError {
```

Comment:
Good call here. The previous error didn't indicate which file was problematic, right?